pkg/ingester/activeseries/model/custom_trackers_config_test.go

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 
-package activeseries
+package activeseriesmodel
```

Comment:
This is in the `model` folder, but the package name is `activeseriesmodel`. Can you make them all consistent please?

Reply:
will do, thank you